### ISSUE TITLE
Graphviz error when using Edge and nodes on latest version 2.47.2

### ISSUE THREAD

**Issue by DaniruKun** [mingrammer/diagrams#536]
When trying to render even a basic diagram with following combination with latest Graphviz, I get the following error:

```
graphviz.backend.CalledProcessError: Command '['dot', '-Kdot', '-Tpng', '-O', 'sample']' died with <Signals.SIGABRT: 6>. [stderr: b'Warning: Orthogonal edges do not currently handle edge labels. Try using xlabels.\nAssertion failed: (sz >= 2), function convertSPtoRoute, file ortho.c, line 155.\n']
```

Steps to reproduce:
1. Environment: 

graphviz: stable **2.47.2**
macOS Big Sur **11.3.1**
Python 3.7 or 3.9

2. Run the following snippet:

```python
from diagrams import Diagram, Edge
from diagrams.onprem.compute import Server
from diagrams.onprem.client import Client

with Diagram("sample"):
    server = Server('server')
    client = Client('client')

    server >> Edge(label='HTTP') << client
```
3. Observer the issue

**Comment by ypicard** [comment]
Yes, broken for me too! Any update on this? 

There is no easy way to install an older version of `graphviz` on mac, brew only has the latest release available.

**Comment by bperlmutter** [comment]
@perryao solution worked for me! hopefully the next stable release of graphviz  will fix this issue so we don't have to use the --HEAD version. 


> I had a similar (but different) error that showed the following on graphviz 2.47.2 installed with homebrew:
> 
> ```
> graphviz.backend.CalledProcessError: Command '['dot', '-Kdot', '-Tpng', '-O', 'foo']' died with <Signals.SIGABRT: 6>. [stderr: b"Warning: node '8c990c8095dc4168b5e430f1d818b1d0', graph 'foo' size too small for label\nAssertion failed: (sz >= 2), function convertSPtoRoute, file ortho.c, line 155.\n"]
> ```
> 
> reinstalling graphviz @ head seems to be a temporary workaround for my issue:
> 
> ```shell
> # homebrew may require you run `brew unlink graphviz`
> brew install graphviz --head
> ```

**Comment by DandyDev** [comment]
I had the same problem (actually saw both error messages) and it was solved for me as well by installing the HEAD version of GraphViz

**Comment by kishvanchee** [comment]
I'm facing the same issue in arch linux. I downgraded graphviz to 2.47.1 and it works now.

**Comment by lamchau** [comment]
Downgrading to 2.47.1 worked for me as well.

For those on macOS using `brew` (`brew install --HEAD graphviz` didn't work for me for some reason)

Homebrew is pretty frustrating for downgrading formulas. Anyhow, here are the steps.

TL;DR
```bash
brew uninstall graphviz &&
curl -O https://raw.githubusercontent.com/Homebrew/homebrew-core/65e5a0ea7548ea9a3adfa99b458d0bf0a2d29f12/Formula/graphviz.rb &&
brew install graphviz.rb &&
rm graphviz.rb
```

Step by step for other people:
```sh
# remove current version
brew uninstall graphviz

# find formula/bottle
# launches url
brew info --github graphviz

# from github url
# 1. https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/graphviz.rb
# 2. Click 'History'
# 3. Locate for the version from @BrewTestBot
# 4. Click '...' > View File > Raw
# 5. Fetch using `cURL`
curl -O https://raw.githubusercontent.com/Homebrew/homebrew-core/65e5a0ea7548ea9a3adfa99b458d0bf0a2d29f12/Formula/graphviz.rb

# install older version
brew install graphviz.rb && rm graphviz.rb

# optional, pin until fix is released
brew pin graphviz
```
